src/spec/integration_support/dummy_cpi.rb

```
@@ -0,0 +1,914 @@
+  end
+  # rubocop:enable ParameterLists
+
+  DELETE_VM_SCHEMA = Membrane::SchemaParser.parse { {vm_cid: String} }
+  def delete_vm(vm_cid)
+    validate_and_record_inputs(DELETE_VM_SCHEMA, __method__, vm_cid)
+    commands.wait_for_unpause_delete_vms
+    detach_disks_attached_to_vm(vm_cid)
+    agent_pid = vm_cid.to_i
+    Process.kill('KILL', agent_pid)
+
+      # rubocop:disable HandleExceptions
```

RuboCop (1.88.0)

[warning] Unnecessary disabling of `HandleExceptions` (unknown cop).

(Lint/RedundantCopDisableDirective)